docs/.vitepress/theme/components/AsideSponsors.vue

```diff
@@ -18,5 +18,71 @@ const sponsors = computed(() => {
 </script>
 
 <template>
+  <a class="viteconf" href="https://viteconf.org">
```

Comment:
Since VitePress opens external links in new tabs, is it good to consistent?
```suggestion
  <a class="viteconf" href="https://viteconf.org" target="_blank">
```

Reply:
Yes, thanks @ydcjeff!